src/Core/Protocol.h

```diff
@@ -62,6 +62,7 @@ const char USER_INTERSERVER_MARKER[] = " INTERSERVER SECRET ";
 
 /// Marker for SSH-keys-based authentication (passed as the user name)
 const char SSH_KEY_AUTHENTICAION_MARKER[] = " SSH KEY AUTHENTICATION ";
+const char JWT_AUTHENTICAION_MARKER[] = " JWT AUTHENTICATION ";
```

Comment:
Please grep all usages of USER_INTERSERVER_MARKER and SSH_KEY_AUTHENTICAION_MARKER (you will find [this place](https://github.com/ClickHouse/ClickHouse/blob/9d8f643f5b306ff02ed8e55dd776afb04e67de49/src/Access/User.cpp#L25-L37))